Formula/monkeysphere.rb

```diff
@@ -48,6 +53,35 @@ def install
     # Consequently, this is a bit of a naughty hack but the least worst option.
     inreplace pkgshare/"keytrans", "#!/usr/bin/perl -T",
                                    "#!/usr/bin/perl -T -I#{libexec}/lib/perl5"
+
+    # find is used with GNU only permissions mode
+    inreplace pkgshare/"common", "$(find ",
+                                 "$(gfind "
+    # use liblockfile
+    inreplace pkgshare/"common", "lockfile-create",
+                                 "dotlockfile"
+    inreplace pkgshare/"common", "lockfile-touch",
+                                 "dotlockfile -T"
+    inreplace pkgshare/"common", "lockfile-remove",
+                                 "dotlockfile -u"
+
+    # Needs BASH 4.0 or later, Apple's BASH 3.2 is insufficient
+    inreplace bin/"monkeysphere", "#!/usr/bin/env bash",
+                                  "#!/usr/local/opt/bash/bin/bash"
```

Comment:
This should be `#{Formula["bash"].opt_bin}/bash`

Reply:
I will make this change.